src/index.ts

```diff
@@ -13,8 +15,9 @@ async function run(): Promise<void> {
     .command("--source", "The file or folder to process")
     .command("--destination", "The output folder")
     .command("--ignores", "A glob list of patterns to ignore")
+    .command("--state", "Which state to render")
```

Comment:
thought: Using a flag her to signify which state is an interesting design choice that may warrant revamp in the future.

Passing the desired state requires knowing the contained states a priori. Perhaps a better method would be to somehow find states and convert the ``state`` flag to be exclusionary?

Reply:
Curious, why wouldn't the user know which states are in their codebase? They are marking up the codebase themselves.

That said, I like the idea of a command that prints a list of available states in a project.

Reply:
To be fair, something like this works

```sh
rg --no-filename -i 'state-start' . | awk 'NF>1{print $NF}' | sort -u
```

It just seems like the tool should output both states in one run, but we'll have to see what typical usage is!